portfolio/src/main/webapp/script.js

```diff
@@ -135,31 +135,51 @@ async function getComments() {
   const response = await fetch('/data' + '?' + 'num-comments=' +
       document.getElementById('num-comments').value);
   const comments = await response.json();
-  const commentsSectionElement = document.getElementById('comments-section');
+  const commentsSectionElement = document.getElementById('comments-list');
   commentsSectionElement.innerHTML = '';
   comments.forEach(function(comm) {
+
     // comment
-    commentsSectionElement.appendChild(
-        createListElement(comm.user + ": " + comm.text + ", " + comm.timestamp 
-        + "; Likes: " + comm.likes + ", Dislikes: " + comm.dislikes));
+    const liElement = document.createElement('li');
+
+    const headerElement = document.createElement('div');
+    headerElement.innerHTML = comm.user.bold() + ', ' + getTimeSince(comm.timestamp);
+    liElement.appendChild(headerElement);
+
+    const likesElement = document.createElement('div');
+    likesElement.id = 'likes-section';
 
     // like button
+    likesElement.appendChild(document.createTextNode(comm.likes));
     var likeButton = document.createElement("button");
     likeButton.innerHTML = ":)";
+    likeButton.id = "like";
     var like = function () {
         editComment(comm.id, "like");
     }
     likeButton.onclick = like;
-    commentsSectionElement.appendChild(likeButton);
+    likesElement.appendChild(likeButton);
+
+    likesElement.appendChild(document.createTextNode(" "));
 
     // dislike button
+    likesElement.appendChild(document.createTextNode(comm.dislikes));
     var dislikeButton = document.createElement("button");
     dislikeButton.innerHTML = ":(";
+    dislikeButton.id = "dislike";
     var dislike = function () {
         editComment(comm.id, "dislike");
     }
     dislikeButton.onclick = dislike;
-    commentsSectionElement.appendChild(dislikeButton);
+    likesElement.appendChild(dislikeButton);
+
+    headerElement.appendChild(likesElement);
+
+    const bodyElement = document.createElement('p');
+    bodyElement.innerHTML = comm.text;
+    liElement.appendChild(bodyElement);
+
+    commentsSectionElement.appendChild(liElement);
```

Comment:
Now that the function body is getting a bit long, maybe it is worth extracting it to a named function.

Reply:
Got it!